### PR TITLE
Default values for functions and flags

### PR DESCRIPTION
**Pull request by WindQAQ**
Address #543

### CODE REVIEW

File: cleverhans_tutorials/mnist_tutorial_jsma.py [hunk 1]
```diff
@@ -25,11 +25,15 @@
 
 FLAGS = flags.FLAGS
 
+NB_EPOCHS = 6
+BATCH_SIZE = 128
+LEARNING_RATE = .001
```
Comment (npapernot) [inline cleverhans_tutorials/mnist_tutorial_jsma.py]: SOURCE_SAMPLES is missing here

File: cleverhans_tutorials/mnist_tutorial_keras_tf.py [hunk 2]
```diff
@@ -27,10 +27,14 @@
 
 FLAGS = flags.FLAGS
 
+NB_EPOCHS = 6
+BATCH_SIZE = 128
+LEARNING_RATE = .001
+
```
Comment (npapernot) [inline cleverhans_tutorials/mnist_tutorial_keras_tf.py]: Why not include TRAIN_DIR and FILENAME as well?

File: cleverhans_tutorials/mnist_tutorial_picklable.py [hunk 3]
```diff
@@ -22,14 +22,19 @@
 
 FLAGS = flags.FLAGS
 
+NB_EPOCHS = 6
+BATCH_SIZE = 128
+LEARNING_RATE = .001
+NB_FILTERS = 64
+
```
Comment (npapernot) [inline cleverhans_tutorials/mnist_tutorial_picklable.py]: Missing CLEAN_TRAIN and BACKPROP_THROUGH_ATTACK ?

File: cleverhans_tutorials/mnist_tutorial_tf.py [hunk 4]
```diff
@@ -25,10 +25,14 @@
 
 FLAGS = flags.FLAGS
 
+NB_EPOCHS = 6
+BATCH_SIZE = 128
+LEARNING_RATE = 0.001
+
```
Comment (npapernot) [inline cleverhans_tutorials/mnist_tutorial_tf.py]: Missing CLEAN_TRAIN and BACKPROP_THROUGH_ATTACK ?

File: cleverhans_tutorials/mnist_tutorial_tfe.py [hunk 5]
```diff
@@ -79,12 +79,12 @@ def attack_selection(attack_string):
 
 
 def mnist_tutorial(train_start=0, train_end=60000, test_start=0,
-                   test_end=10000, nb_epochs=6, batch_size=128,
-                   learning_rate=0.001,
+                   test_end=10000, nb_epochs=NB_EPOCHS, batch_size=BATCH_SIZE,
```
Comment (npapernot) [inline cleverhans_tutorials/mnist_tutorial_tfe.py]: Where are these vars declared?

**Comment by WindQAQ** [comment]
Sorry for missing lots of values.... There are too many of them. Please check it out. Thx!

**Comment by goodfeli** [comment]
Thanks!

**Pull request merged**